filesystems/drs/src/test/scala/cromwell/filesystems/drs/DrsPathBuilderSpec.scala

```diff
@@ -277,27 +293,79 @@ class DrsPathBuilderSpec extends TestKitSuite with AnyFlatSpecLike with Matchers
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false),
+      isAbsolute = false,
+    ),
 
     GoodPath(
       description = "an absolute path without a host",
       path = s"drs://blah",
       normalize = false,
-      pathAsString = s"drs://blah/",
-      pathWithoutScheme = s"blah/",
+      pathAsString = s"drs://blah",
+      pathWithoutScheme = s"blah",
       parent = null,
       getParent = null,
-      root = s"drs://blah/",
+      root = s"drs://blah",
       name = "",
       getFileName = null,
       getNameCount = 1,
-      isAbsolute = false)
+      isAbsolute = false,
+    ),
+
+    // No spec says this is illegal... so pass it to Martha's various GCFs JIC
+    GoodPath(
+      description = "a bucketless path",
+      path = s"drs://",
+      normalize = false,
+      pathAsString = s"drs://",
+      pathWithoutScheme = "",
+      parent = null,
+      getParent = null,
+      root = s"drs://",
+      name = "",
+      getFileName = null,
+      getNameCount = 1,
+      isAbsolute = false,
+    ),
+
+    // https://docs.google.com/document/d/1Wf4enSGOEXD5_AE-uzLoYqjIp5MnePbZ6kYTVFp1WoM/edit
```

Comment:
```suggestion
    // A DRS URL which might ultimately resolve to https://docs.google.com/document/d/1Wf4enSGOEXD5_AE-uzLoYqjIp5MnePbZ6kYTVFp1WoM/edit
```

Reply:
Thanks-- will update the comment. It doesn't resolve to the doc, it originates from there.